packages/components/src/components/hds/application-state/header.hbs

```diff
@@ -3,18 +3,18 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 <div class="hds-application-state__header" ...attributes>
-  {{#if @icon}}
-    <div class="hds-application-state__icon">
-      <FlightIcon @name={{@icon}} @size="24" />
-    </div>
-  {{/if}}
-  <Hds::Text::Display class="hds-application-state__title" @tag="div" @size="400" @weight="semibold">
-    {{@title}}
-  </Hds::Text::Display>
   {{#if @errorCode}}
     <Hds::Text::Body class="hds-application-state__error-code" @tag="div" @size="100" @weight="medium">
```

Comment:
Did you forget to check the styles in Figma? 🤔
```suggestion
    <Hds::Text::Body class="hds-application-state__error-code" @tag="div" @size="100" @weight="medium" @color="foreground-faint">
```

Reply:
![Screenshot 2024-07-25 at 10 30 32 AM](https://github.com/user-attachments/assets/f2ee1829-43b3-4301-918a-225717422d66)

Was relying on this in the mocks, but I'll check individual elements in the future to be sure nothing has changed.

Reply:
Yes, always better to check directly the actual Figma components and their inner parts:
- text (size, weight, color)
- icon (size, color)
- spacing between elements
- alignments
- wrapping